aws/sdk/build.gradle.kts

```diff
@@ -360,7 +360,7 @@ tasks.register<ExecRustBuildTool>("hydrateReadme") {
     binaryName = "publisher"
     arguments = listOf(
         "hydrate-readme",
-        "--versions-manifest", outputDir.resolve("versions.toml").toString(),
+        "--versions-manifest", sdkOutputDir.resolve("versions.toml").toString(),
```

Comment:
This path seems wrong. The `versions.toml` resides in the aws-sdk-rust repo root rather than in `sdk/`.

Reply:
Ah, thanks for catching. `publisher`'s subcommand `generate-version-manifest` needs to be able to specify different directories for input and output. Addressed in 7af15c5 and 7d54876.